core/src/rack.c

```diff
@@ -86,11 +86,8 @@ int set_rack_to_string(Rack *rack, const char *rack_string,
   reset_rack(rack);
 
   uint8_t mls[MAX_BAG_SIZE];
-  int num_mls =
-      str_to_machine_letters(letter_distribution, rack_string, false, mls);
-  if (num_mls > MAX_BAG_SIZE) {
-    log_fatal("rack overflow: %d\n", num_mls);
-  }
+  int num_mls = str_to_machine_letters(letter_distribution, rack_string, false,
+                                       mls, MAX_BAG_SIZE);
```

Comment:
is it intended to no longer [`abort()`](https://github.com/jvc56/MAGPIE/blob/c52c7aa5810dc15063658c4534eef3c6bca7b1bc/core/src/log.c#L159) on invalid input?

Reply:
Yeah I don't think we should abort for overflow or any other failure. Checking if rack was set is the callers responsibility.